webapp/src/websocket_handlers.ts

```diff
@@ -590,45 +587,38 @@ export function handleHostRemoved(store: Store, ev: WebSocketMessage<HostControl
     }
 
     const sessionID = client.getSessionID();
-    if (ev.data.session_id !== sessionID) {
-        const profile = profilesInCurrentCallMap(store.getState())[ev.data.user_id] ||
-            getUser(store.getState(), ev.data.user_id);
-        if (!profile) {
-            return;
-        }
-
-        const displayName = getUserDisplayName(profile);
-
-        const hostNotice: HostControlNotice = {
-            type: HostControlNoticeType.HostRemoved,
-            callID: ev.data.call_id,
-            noticeID: generateId(),
-            displayName,
-        };
-
-        store.dispatch({
-            type: HOST_CONTROL_NOTICE,
-            data: hostNotice,
-        });
-
-        setTimeout(() => {
-            store.dispatch({
-                type: HOST_CONTROL_NOTICE_TIMEOUT_EVENT,
-                data: {
-                    callID: ev.data.call_id,
-                    noticeID: hostNotice.noticeID,
-                },
-            });
-        }, HOST_CONTROL_NOTICE_TIMEOUT);
+    if (ev.data.session_id === sessionID) {
+        getCallsClient()?.disconnect(new Error(hostRemovedMsg));
```

Comment:
🙉

Reply:
what happened here :stuck_out_tongue:

Reply:
This is the correct way to do it. Should have done it earlier. :)